examples/ios/cubic-demo/CubicAudioRecorder.swift

```diff
@@ -0,0 +1,206 @@
+//
+//  CubicDemoManager.swift
```

Comment:
This filename doesn't match with the comment name.

Reply:
Also, since we discussed the app will be in it's own repo, please remove from here.